### ISSUE TITLE
Extension host terminated unexpectedly. Please reload the window to recover.

### ISSUE THREAD

**Issue by johnashu** [Microsoft/vscode#33451]
- VSCode Version: 1.15.1
- OS Version: windows 10

Steps to Reproduce:

1.  install 64 bit VS Code
2. Open a file

Reproduces without extensions: Yes

having uninstalled and cleaned this system and registry and disabled extensions.. I am at a loss..

Extension host terminated unexpectedly. Please reload the window to recover.
e.doShow @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:28
/C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:29 Extension host terminated unexpectedly. Code:  3221225477  Signal:  null
t.onExit @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:29

**Comment by alexandrudima** [comment]
@johnashu 

The exit code 3221225477 is the Windows error code `C0000005` or `ACCESS_VIOLATION`.



This unfortunately doesn't tell us a lot about the root cause of the crash, except that it happens in native code. At this time, we also don't have any easy way to extract the memory dump / crash data from an extension host process (@ramya-rao-a perhaps we could have a command line flag to enable this for such troubleshooting purposes?). To find out if this is a systematic problem, may I kindly ask you that you try out downloading the following zip versions of VS Code:

* [1.15.1 - 32 bit](https://az764295.vo.msecnd.net/stable/41abd21afdf7424c89319ee7cb0445cc6f376959/VSCode-win32-ia32-1.15.1.zip)
* [1.15.1 - 64 bit](https://az764295.vo.msecnd.net/stable/41abd21afdf7424c89319ee7cb0445cc6f376959/VSCode-win32-x64-1.15.1.zip)
* [1.16.0-insider (30.08.2017) - 32 bit](https://az764295.vo.msecnd.net/insider/34cc9f2209be766d5be30a6ecdbb41f5e81fd56b/VSCode-win32-ia32-1.16.0-insider.zip)
* [1.16.0-insider (30.08.2017) - 64 bit](https://az764295.vo.msecnd.net/insider/34cc9f2209be766d5be30a6ecdbb41f5e81fd56b/VSCode-win32-x64-1.16.0-insider.zip)

You can try launching them one at a time. Is it that in all cases the extension host crashes on startup? If the extension host process starts up successfully and crashes only after certain actions, have you found what those actions are?

**Comment by johnashu** [comment]
@alexandrudima 

I just checked again and I realised that when I opened code using

code --disable-extensions

I actually opened it with this command

code --disable<space>-extensions

With extensions disabled it runs fine..

I will troubleshoot my extensions and create a new issue if I get stuck.

I will attempt to post the offender if I can successfully catch the problem extension

Thanks again!
